russian-gost-r-705-2008-numeric-citations-alphabetical-bibliography.csl

```diff
@@ -0,0 +1,175 @@
+    <title-short>GOST R (numeric-citations, alphabetical-bibliography)</title-short>
+    <id>http://www.zotero.org/styles/russian-gost-r-705-2008-numeric-citations-alphabetical-bibliography</id>
+    <link href="http://www.zotero.org/styles/russian-gost-r-705-2008-numeric-citations-alphabetical-bibliography" rel="self"/>
+    <link href="http://protect.gost.ru/document.aspx?control=7&amp;id=173511" rel="documentation"/>
+    <link href="http://standartgost.ru/g/%D0%93%D0%9E%D0%A1%D0%A2_%D0%A0_7.0.5-2008" rel="documentation"/>
+    <author>
+      <name>Rybakov Dmitriy</name>
+    </author>
+    <category citation-format="numeric"/>
+    <category field="generic-base"/>
+    <summary>Russian GOST-2008 style. Book, webpage and article chapters edited. Note!!! In articles form of № = Issue (Volume).</summary>
+    <updated>2015-06-27T09:14:23+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <macro name="editor">
+    <names variable="editor" delimiter=", " suffix=",">
+      <label prefix=" " suffix=" "/>
+      <name initialize-with="."/>
+    </names>
+  </macro>
+  <macro name="author">
+    <names variable="author" delimiter="," prefix=" / " suffix=",">
+      <name et-al-use-first="5" initialize-with="."/>
+      <label form="short" strip-periods="true"/>
+    </names>
+  </macro>
+  <macro name="access">
+    <group>
+      <text value="[Электронный ресурс]. "/>
+      <text variable="URL" prefix="URL: "/>
+      <group prefix=" (" suffix=").">
+        <text term="accessed" suffix=": "/>
+        <date variable="accessed">
+          <date-part name="day" suffix="." form="numeric-leading-zeros"/>
+          <date-part name="month" suffix="." form="numeric-leading-zeros"/>
+          <date-part name="year"/>
+        </date>
+      </group>
+    </group>
+  </macro>
+  <macro name="title">
+    <text variable="title"/>
+  </macro>
+  <macro name="translator">
+    <names variable="translator" delimiter="," suffix=",">
+      <label prefix=" " suffix=" "/>
+      <name initialize-with="."/>
+    </names>
+  </macro>
+  <macro name="publisher">
+    <group suffix=",">
+      <text variable="publisher-place" suffix=":"/>
+      <text variable="publisher" prefix=" "/>
+    </group>
+    <text macro="year-date" prefix=" " suffix="."/>
+  </macro>
+  <macro name="year-date">
+    <date variable="issued">
+      <date-part name="year"/>
+    </date>
+  </macro>
+  <macro name="language-sort">
+    <text variable="note"/>
+    <text value="Z999"/>
+  </macro>
+  <macro name="citation-number">
+    <text variable="citation-number" suffix=". "/>
+  </macro>
+  <macro name="author-first">
+    <names variable="author">
+      <name delimiter-precedes-last="always" et-al-min="4" et-al-use-first="1" initialize-with="." name-as-sort-order="all" sort-separator=" "/>
```

Comment:
remove et-al-min="4" et-al-use-first="1"  -- it's already set at a higher level.